lib/utils/findAnimationName.js

```diff
@@ -63,5 +67,6 @@ module.exports = function findAnimationName(value) {
 		animationNames.push(valueNode);
 	});
 
-	return animationNames;
+	// Cannot find animation name if shorthand has non-standard syntax value (#5532)
+	return hasNonStandardSyntaxValue ? [] : animationNames;
```

Comment:
[suggest] How about the following refactorings?

- `hasNonStandardSyntaxValue` is enough now, but it could be flexible for generic ignoring logic.
- We can omit `animationNames.push()` if non-standard syntax.
- We can simplify `return animationNames` by clearing the array.

```js
let shouldBeIgnored = false;

valueNodes.walk((valueNode) => {
  if (shouldBeIgnored) return;

  // ...

  if (!isStandardSyntaxValue(valueLowerCase)) {
    shouldBeIgnored = true;
    animationNames.length = 0; // clear
    return;
  }
});

return animationNames;
```

Reply:
Good suggestion! I'll update the PR once I'm home!